src/libs/pipeline_configuration.py

```diff
@@ -0,0 +1,283 @@
+                    run_id_field="rqa_s01e04_run_id",
+                    analysis_file_key="rqa_s01e04_",
+                    cleaner=None,
+                    code_scheme=CodeSchemes.ICRAF_S01E04),
+                    
+        CodingPlan(raw_field="rqa_s01e05_raw",
+                    coded_field="rqa_so1e05_coded",
+                    time_field="sent_on",
+                    coda_filename="icraf_s01e05.json",
+                    icr_filename="icraf_s01e05.csv",
+                    run_id_field="rqa_s01e05_run_id",
+                    analysis_file_key="rqa_s01e05_",
+                    cleaner=None,
+                    code_scheme=CodeSchemes.ICRAF_S01E05,
+                    binary_code_scheme=CodeSchemes.YES_NO,
+                    binary_coded_field="rqa_s01e05_yes_no_coded",
+                    binary_analysis_file_key="rqa_s01e05_yes_no"),
+        
+        CodingPlan(raw_field="rqa_s01e06_raw",
+                    coded_field="rqa_so1e06_coded",
+                    time_field="sent_on",
+                    coda_filename="icraf_s01e06.json",
+                    icr_filename="icraf_s01e06.csv",
+                    run_id_field="rqa_s01e06_run_id",
+                    analysis_file_key="rqa_s01e06_",
+                    cleaner=None,
+                    code_scheme=CodeSchemes.ICRAF_S01E06),
+
+        CodingPlan(raw_field="current_practices_raw",
+                    coded_field="current_practices_coded",
+                    time_field="current_practices_time",
+                    coda_filename="current_practices.json",
+                    analysis_file_key="current_practices_",
+                    cleaner=None,
+                    code_scheme=CodeSchemes.CURRENT_PRACTICES),
+        
+        CodingPlan(raw_field="upper_tana_practices_raw",
+                    coded_field="upper_tana_practices_coded",
+                    time_field="upper_tana_practices_time",
+                    coda_filename="upper_tana_practices.json",
+                    analysis_file_key="upper_tana_practices_",
+                    cleaner=None,
+                    code_scheme=CodeSchemes.UPPER_TANA_PRACTISES),
+        
+        CodingPlan(raw_field="new_practices_raw",
+                    coded_field="new_practices_coded",
+                    time_field="new_practices_time",
+                    coda_filename="new_practices.json",
+                    analysis_file_key="new_practices_",
+                    cleaner=None,
+                    code_scheme=CodeSchemes.NEW_PRACTICES),
+        
+        CodingPlan(raw_field="new_practices_reasons_raw",
+                    coded_field="new_practices_reasons_coded",
+                    time_field="new_practices_reasons_time",
+                    coda_filename="new_practices_reasons.json",
+                    analysis_file_key="new_practices_reasons_",
+                    cleaner=None,
+                    code_scheme=CodeSchemes.REASONS_FOR_NEW_PRACTICES),
```

Comment:
Might be worth renaming this scheme to bring it in line with the field names.

Reply:
Renamed to NEW_PRACTICES_CHALLENGES